src/components/PositionsList/PositionItem/PositionItem.tsx

```diff
@@ -47,6 +49,33 @@ export const PositionItem: React.FC<IPositionItem> = ({
     initialXtoY(tickerToAddress(tokenXName), tickerToAddress(tokenYName))
   )
 
+  const getPercentageRatio = () => {
+    if (maxTick <= currentTick) {
```

Comment:
Is the `currentTick` here the tick taken directly from the state pool? If so, these percentage results will be very inaccurate. It's better to take the `currentSqrtPprice` from the pool and calculate the tick index